firefox-esr.txt

```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Mozilla Firefox 52 ESR
+Version = 52.9.0
+License = MPL/GPL/LGPL
+Description = A popular Web browser. Free and open source, made by a global community.
```

Comment:
It may make sense to add a sentence `Scores xxx/555 on https://html5test.opensuse.org` where xxx would have to be tested though.

Reply:
Maybe in the later PR, once and for all.